panacea/account.go

```diff
@@ -43,7 +43,7 @@ func (oa OracleAccount) GetAddress() string {
 }
 
 func (oa OracleAccount) AccAddressFromBech32() sdk.AccAddress {
-	return oa.pubKey.Bytes()
+	return oa.pubKey.Address().Bytes()
```

Comment:
We shouldn't use AccAddress directly from the PublicKey, but rather call the Address function.